rust/pact_verifier/src/lib.rs

```diff
@@ -1182,11 +1257,15 @@ pub async fn verify_pact_internal<'a, F: RequestFilterExecutor, S: ProviderState
       }
     };
 
+    // TODO: Update this to use V4 models
```

Comment:
Is this something that needs addressing in this PR, or a reminder for cleanup in the future (i.e. no functional impact)?

Reply:
I decided to make it tech debt. Possibly just to annoy Beth.

Reply:
It's only tech debt if Matt made you do it.